src/test/java/org/cactoos/scalar/NumberOfTest.java

```diff
@@ -98,7 +97,7 @@ public void failsIfTextDoesNotRepresentADouble() throws IOException {
     @Test
     public void parsesValueInt() throws IOException {
         new Assertion<>(
-            "Can't parse into int",
+            "Must parse into int",
             () -> new NumberOf("185").value().intValue(),
```

Comment:
@marceloamadeu please simplify this to be like the other tests (without using this lambda and `ScalarHasValue`)